pcli/src/command/tx.rs

```diff
@@ -808,6 +830,63 @@ impl TxCmd {
                     .await?;
                 app.build_and_submit_transaction(plan).await?;
             }
+            TxCmd::Ics20Withdrawal {
+                destination_chain_id,
+                destination_chain_address,
+                amount,
+                denom,
+                timeout_height,
+                timeout_timestamp,
+                source_channel,
+                return_address_index,
+            } => {
+                let fee = Fee::from_staking_token_amount(1_000_000u64.into());
```

Comment:
This should probably default to `0` rather than `1` -- since we're going to do a fee overhaul as part of distributions anyways, I think we don't need to worry about setting it for now.